authentication/urls.py

```diff
@@ -1,8 +1,15 @@
 from django.urls import path
-from authentication.views import *
+from authentication.views import RegisterView, LoginView, SendVerificationEmailView, SendRecoverPasswordEmailView
+from rest_framework_simplejwt.views import TokenObtainPairView
+from rest_framework_simplejwt.views import TokenRefreshView
 
 app_name = 'authentication'
 
 urlpatterns = [
-    path('try', returnTodoJson, name='returnTodoJson'),
+    path('register/', RegisterView.as_view(), name='register'),
+    path('login/', LoginView.as_view(), name='login'),
+    path('token/', TokenObtainPairView.as_view(), name='token_obtain_pair'),
```

Comment:
Personally, I don't think this is necessary. I think the best apporach is to send token after POST request on login endpoint (which you already did), so probably just remove this one.

We will just need to use refresh token endpoint to refresh our access token (which I expect refresh token endpoint will send the access token back to us).

Reply:
Ok, fix it real soon